resources/protocols/intro-to-disinfection-and-sterilization.md

```diff
@@ -273,26 +273,36 @@ Common enzymatic detergents used in body art settings include Alconox Tergazyme,
 
 #### What Ultrasonic Cleaning Does and Doesn’t Do
 
-<div class="comparison-grid">
-  <div>
-    <h5>✅ Does</h5>
-    <ul>
-      <li>Removes debris from hard-to-reach areas</li>
-      <li>Enhances manual cleaning efficacy</li>
-      <li>Standardizes the cleaning process</li>
-      <li>Reduces handling of contaminated instruments</li>
-      <li>Improves efficiency of the cleaning process</li>
-    </ul>
-  </div>
-  <div>
-    <h5>❌ Doesn’t</h5>
-    <ul>
-      <li>Sterilize instruments</li>
-      <li>Disinfect instruments</li>
-      <li>Replace manual pre-cleaning for heavily soiled items</li>
-      <li>Remove all contaminants without proper solution and time</li>
-    </ul>
-  </div>
+<div class="table-responsive">
```

Comment:
This can be a borderless table and then we get all the layout benefits of tables, but it doesn't look like a table.